packages/api/models/granules.js

```diff
@@ -35,26 +35,28 @@ class Granule extends Manager {
     stepFunctionUtils = StepFunctions,
     cmrUtils = cmrjsCmrUtils,
   } = {}) {
-    const globalSecondaryIndexes = [{
-      IndexName: 'collectionId-granuleId-index',
-      KeySchema: [
-        {
-          AttributeName: 'collectionId',
-          KeyType: 'HASH',
+    const globalSecondaryIndexes = [
+      {
+        IndexName: 'collectionId-granuleId-index',
+        KeySchema: [
+          {
+            AttributeName: 'collectionId',
+            KeyType: 'HASH',
+          },
+          {
+            AttributeName: 'granuleId',
+            KeyType: 'RANGE',
+          },
+        ],
+        Projection: {
+          ProjectionType: 'ALL',
         },
-        {
-          AttributeName: 'granuleId',
-          KeyType: 'RANGE',
```

Comment:
What's up with this removal?

Reply:
It's actually *not* a removal, it's the result of a prettier refactor (note the [{ to [\n{ change.    Thanks opinionated linter!